velox/exec/tests/SparkAggregationFuzzerTest.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <folly/init/Init.h>
+#include <gflags/gflags.h>
+#include <gtest/gtest.h>
+#include <unordered_set>
+
+#include "velox/exec/tests/AggregationFuzzerRunner.h"
+#include "velox/functions/sparksql/aggregates/Register.h"
+
+DEFINE_int64(
+    seed,
+    0,
+    "Initial seed for random number generator used to reproduce previous "
+    "results (0 means start with random seed).");
+
+DEFINE_string(
+    only,
+    "",
+    "If specified, Fuzzer will only choose functions from "
+    "this comma separated list of function names "
+    "(e.g: --only \"min\" or --only \"sum,avg\").");
+
+int main(int argc, char** argv) {
+  facebook::velox::functions::sparksql::aggregates::registerAggregateFunctions(
+      "");
+
+  ::testing::InitGoogleTest(&argc, argv);
+
+  // Calls common init functions in the necessary order, initializing
+  // singletons, installing proper signal handlers for better debugging
+  // experience, and initialize glog and gflags.
+  folly::init(&argc, &argv);
+
+  // TODO: List of the functions that at some point crash or fail and need to
+  // be fixed before we can enable.
+  std::unordered_set<std::string> skipFunctions = {
+      "last" // Not support TIMESTAMP.
```

Comment:
Maybe create GitHub issue with fuzzer-found label and add a link to it here? Would you be willing to help fix this problem?

Reply:
Sure, I will create issue soon with fuzzer test error log, and try to fix it.
Let's skip it for now.

Reply:
See #4482 .